store/sqlstore/post_store.go

```diff
@@ -2139,6 +2139,13 @@ func (s *SqlPostStore) AnalyticsPostCount(options *model.PostCountOptions) (int6
 			Where(sq.Eq{"c.TeamId": options.TeamId})
 	}
 
+	if options.UsersPostsOnly {
+		query = query.Where(sq.And{
+			sq.Eq{"p.Type": ""},
+			sq.Expr("p.UserId NOT IN (SELECT Userid FROM Bots)"),
```

Comment:
```suggestion
			sq.Expr("p.UserId NOT IN (SELECT UserId FROM Bots)"),
```

Reply:
Done. 
Although it does raise an interesting point, that to work with MySql, the Columns casing is not "mandatory" to be correct. I think it was mentioned somewhere that Squirrel needs to have correct casing to work with MySql.